### PR TITLE
Register page design

### PR DESCRIPTION
**Pull request by sruthij1**

### CODE REVIEW

File: templates/register.html [hunk 1]
```diff
@@ -1,59 +1,77 @@
...
+    <!--script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
   <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script-->
-  <script src="https://www.google.com/recaptcha/api.js" async defer></script>
-</head>
-<style>
-.center_div{
-    margin: 0 auto;
-    width:30% /* value of your choice which suits your alignment */
-}
+    <script src="https://www.google.com/recaptcha/api.js" async defer></script>
```
Comment (ntjandra) [inline templates/register.html]: What is async defer?

File: templates/register.html [hunk 2]
```diff
@@ -1,59 +1,77 @@
...
-      <input type="password" class="form-control" id="pwd" placeholder="Enter password" style="width: 250px" name="pwd">
+  <body>
+    <div class="container_center_div">
+      <h1 align="center">
+        <font face="Trebuchet MS"><b>Create An Account</b></font>
+      </h1>
+      <br />
+      <form action="/action_page.php">
+        <div class="form-group">
+          <label for="usrname">Username:</label>
```
Comment (ntjandra) [inline templates/register.html]: Same typo

File: templates/register.html [hunk 3]
```diff
@@ -1,59 +1,77 @@
...
+          <input
+            type="text"
+            class="form-control"
+            id="usrname"
+            placeholder="Enter username"
+            style="width: 250px"
+            name="usrname"
```
Comment (ntjandra) [inline templates/register.html]: Typo here as well.

File: templates/register.html [hunk 4]
```diff
@@ -1,59 +1,77 @@
...
+        <font face="Trebuchet MS"><b>Create An Account</b></font>
+      </h1>
+      <br />
+      <form action="/action_page.php">
+        <div class="form-group">
+          <label for="usrname">Username:</label>
+          <input
+            type="text"
+            class="form-control"
+            id="usrname"
```
Comment (ntjandra) [inline templates/register.html]: Id should not be this.

File: templates/register.html [hunk 5]
```diff
@@ -1,59 +1,77 @@
...
+            type="password"
+            class="form-control"
+            id="pwd"
+            placeholder="Enter password"
+            style="width: 250px"
+            name="pwd"
+          />
+        </div>
+        <div class="form-group">
+          <label for="pwdConf">Confirm Password:</label>
```
Comment (ntjandra) [inline templates/register.html]: Why is there a : in the label

File: templates/register.html [hunk 6]
```diff
@@ -1,59 +1,77 @@
...
+            id="pwdConf"
+            placeholder="Confirm password"
+            style="width: 250px"
+            name="pwdConf"
+          />
+        </div>
+        <form action="?" method="POST">
+          <div
+            class="g-recaptcha"
+            data-sitekey="6Lf0_b4UAAAAAKZX3E_Ec_bqVre6B6Xpn_3Vfd-b"
```
Comment (ntjandra) [inline templates/register.html]: Oh No. Not this again. Please use Environment variables.

**Comment by ntjandra** [comment]
React Components should be .js not .html
